tslint.json

```diff
@@ -1,9 +1,9 @@
 {
 	"extends": "tslint:recommended",
 	"rules": {
-    "ban-types": false,
-    "interface-name" : [true, "never-prefix"],
-    "interface-over-type-literal": false,
+        "ban-types": false,
+        "interface-name" : [true, "never-prefix"],
+        "interface-over-type-literal": false,
 		"indent": [ true, "tabs" ],
```

Comment:
While we're in here:
Setting this to `"indent": [true, "tabs", 4]` will allow the linter to auto-fix this issue rather than just complaining and making me manually convert to tabs every time I build :P
 https://palantir.github.io/tslint/rules/indent/